Week3/homework/step2-5.js

```diff
@@ -3,10 +3,11 @@
 function multiplyAll(arr) {
   // eslint-disable-next-line
   let product = 1;
-
-  // Replace this comment and the next line with your code
-  console.log(arr, product);
-
+  arr.forEach(element => {
+    element.forEach(item => {
+      product *= item;
```

Comment:
Good! I like that you've used `forEach` here. Maybe the param names could be more descriptive - as `element` and `item` are a bit vague to me. Maybe `innerArray`  and `innerArrayItem`.

Reply:
innerArrayItem is great name